Features/docs facade attributes (#165)
oemof/oemof-thermal: docs/getting_started.rst

To help setting up more detailed components in a simple way, oemof.thermal provides facades based on the
`oemof.tabular.facades module <https://oemof-tabular.readthedocs.io/en/stable/reference/oemof.tabular.html>`_.
Facades are classes that offer a simpler interface to more complex classes. More specifically, the :class:`Facade` s
in this module inherit from `oemof.solph`'s generic classes to serve as more concrete and energy specific interface.
The concept of the facades has been derived from oemof.tabular. The idea is to be able to
instantiate a :class:`Facade` using only keyword arguments. Under the hood the :class:`Facade` then
uses these arguments to construct an `oemof.solph` component and sets it up to be easily used in an
:py:obj:`EnergySystem`. Usually, a subset of the attributes of the parent class remains while another

> part can be addressed by more specific or simpler attributes. In oemof.thermal each technology has a facade class,
> that can be found in the module oemof.thermal.facades. See the

Author: ```suggestion
part can be addressed by more specific or simpler attributes. In oemof.thermal, some of the technologies hava a facade class
that can be found in the module oemof.thermal.facades. See the
```

Author: There is no facade for the heatpump in oemof.thermal, e.g.

Reviewer: Oh yes, you are right. Then this text was outdated anyway..
